mutiny-wasm/src/models.rs

```diff
@@ -338,3 +342,90 @@ impl AuthProfile {
         JsValue::from_serde(&serde_json::to_value(&self.used_services).unwrap()).unwrap()
     }
 }
+
+#[derive(Debug, Serialize, Deserialize, Clone, Copy, Eq, PartialEq, Hash)]
+#[wasm_bindgen]
+pub enum TagKing {
+    Label,
+    Contact,
+}
+
+#[derive(Debug, Serialize, Deserialize, Clone, Eq, PartialEq)]
+#[wasm_bindgen]
+pub struct TagItem {
+    id: String,
+    pub kind: TagKing,
```

Comment:
fwiw I really appreciate this kind of enum in js land... a property of a big object with a bunch of options, rather than multiple object types. I know it makes more work in rust land but it's easier for me to work with

Reply:
Yeah that was my goal, made the nice enum in our internal stuff but struct with lots of `Option`s for the wasm crate.